api/apps/api/test/access-control/project-acl-get.e2e-spec.ts

```diff
@@ -0,0 +1,132 @@
+  const projectId = await fixtures.GivenProjectWasCreated();
+  let response = await fixtures.WhenAddingANewViewerToTheProjectAsOwner(
+    projectId,
+  );
+  fixtures.ThenNoContentIsReturned(response);
+  response = await fixtures.WhenAddingANewContributorToTheProjectAsOwner(
+    projectId,
+  );
+  fixtures.ThenNoContentIsReturned(response);
+  response = await fixtures.WhenAddingANewOwnerToTheProjectAsOwner(projectId);
+  fixtures.ThenNoContentIsReturned(response);
+
+  response = await fixtures.WhenGettingProjectUsersAsOwner(projectId);
+  fixtures.ThenAllUsersinProjectAfterEveryTypeOfUserHasBeenAddedAreReturned(
+    response,
+  );
+});
+
+test(`add every type of user to a project as a project contributor`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  await fixtures.WhenAddingANewContributorToTheProjectAsOwner(projectId);
+
+  let response = await fixtures.WhenAddingANewViewerToTheProjectAsContributor(
+    projectId,
+  );
+  fixtures.ThenForbiddenIsReturned(response);
+  response = await fixtures.WhenAddingANewContributorToTheProjectAsContributor(
+    projectId,
+  );
+  fixtures.ThenForbiddenIsReturned(response);
+  response = await fixtures.WhenAddingANewOwnerToTheProjectAsContributor(
+    projectId,
+  );
+  fixtures.ThenForbiddenIsReturned(response);
+});
+
+test(`add every type of user to a project as a project viewer`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  await fixtures.WhenAddingANewContributorToTheProjectAsOwner(projectId);
+
+  let response = await fixtures.WhenAddingANewViewerToTheProjectAsViewer(
+    projectId,
+  );
+  fixtures.ThenForbiddenIsReturned(response);
+  response = await fixtures.WhenAddingANewContributorToTheProjectAsViewer(
+    projectId,
+  );
+  fixtures.ThenForbiddenIsReturned(response);
+  response = await fixtures.WhenAddingANewOwnerToTheProjectAsViewer(projectId);
+  fixtures.ThenForbiddenIsReturned(response);
+});
+
+test(`delete every type of user from the project as owner`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  await fixtures.WhenAddingANewViewerToTheProjectAsOwner(projectId);
+  await fixtures.WhenAddingANewContributorToTheProjectAsOwner(projectId);
+  await fixtures.WhenAddingANewOwnerToTheProjectAsOwner(projectId);
+  let revokeResponse = await fixtures.WhenRevokingAccessToViewerFromProjectAsOwner(
+    projectId,
+  );
+  fixtures.ThenNoContentIsReturned(revokeResponse);
+  revokeResponse = await fixtures.WhenRevokingAccessToContributorFromProjectAsOwner(
+    projectId,
+  );
+  fixtures.ThenNoContentIsReturned(revokeResponse);
+  revokeResponse = await fixtures.WhenRevokingAccessToOwnerFromProjectAsOwner(
+    projectId,
+  );
+  fixtures.ThenNoContentIsReturned(revokeResponse);
+  const response = await fixtures.WhenGettingProjectUsersAsOwner(projectId);
+  fixtures.ThenSingleOwnerUserInProjectIsReturned(response);
+});
```

Comment:
```
fixtures.GivenProjectWasCreated();
const viewerId = fixtures.GivenUserInProject(projectId, `viewer`);
fixtures.GivenUserInProject(projectId, `contributor`);
fixtures.GivenUserInProject(projectId, `owner`);

fixtures.WhenRevokingAccess(projectId, viewerId /* or other...*/ );

fixtures.Then.... /** (get & verify under the hood) **/
```


also would recommend to split those tests (if there are too much within single file, split into multiple files - there, `project-acl-revoke`)

Reply:
Same thing as https://github.com/Vizzuality/marxan-cloud/pull/640#discussion_r757509916